com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalTarget.cs

```diff
@@ -1479,6 +1695,19 @@ static class CoreKeywordDescriptors
             scope = KeywordScope.Global,
         };
 
+        public static readonly KeywordDescriptor SceneSelectionPass = new KeywordDescriptor()
+        {
+            displayName = "Scene Selection Pass",
+            referenceName = "SCENESELECTIONPASS",
+            type = KeywordType.Boolean,
```

Comment:
Scope defaults to Local. Should the scope be local or global? local is typically set from material, while global set typically set from cmdbuf.
Definition defaults to ShaderFeature. Typically useful for materials.

Reply:
Uh good question. HDTarget has these explicitly as:
```
definition = KeywordDefinition.ShaderFeature,
scope = KeywordScope.Local,
```
So I'd assume we'd want the same? I'm honestly not very clear on the difference

Reply:
fyi, this doesn't matter for trunk as now all keywords are local, both `shader_feature` and `shader_feature_local` map to local keywords and `_local` will be deprecated. https://unity.slack.com/archives/G160VS3HD/p1619621386214200

Reply:
Yeah, I think it's good enough since it works for HDRP too.